src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -289,14 +289,24 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if (issuanceMintAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Mint tokens to address
-        _mint(_receiver, issuanceMintAmount);
+
+        // Use abstract function to distribute issuance tokens
```

Comment:
The term "abstract function" doesn't exist in Solidity. Do you mean to say that the `virtual` function is used, as it can be overridden by a downstream contract potentially?

Reply:
Will use virtual

Reply:
Done